assignments/lambda-classes.js

```diff
@@ -54,10 +54,10 @@ class ProjectManager extends Instructor {
         this.gradClassName = props.gradClassName;
         this.favInstructor = props.favInstructor;
     }
-    standUp () {
+    standUp (channel) {
         return `${this.name} announces to ${channel}, @channel standy times!`;
     }
-    debugsCode () { // STILL MISSING STUDENT OBJECT ========================================
+    debugsCode (student.name, subject) { // STILL MISSING STUDENT OBJECT ========================================
```

Comment:
Hey, Luis! Great job on this assignment! Most everything is working however on line 60 you have a small error. If you refer to line 26 I think you might spot the issue. Your code was very easy to read. Your comments were awesome!!